### ISSUE TITLE
Defining Custom Views - getTrianglePath() allocating new Points in onDraw()

### ISSUE THREAD

**Issue by teotigraphix** [codepath/android_guides#313]
This is a no-no and it's not caught by the compiler(IDE) because it's in a caller method.

Just some best practices, especially for a custom view guide.

(If I had the time I would fix it but just wanted to note this.)

**Comment by DevHossamHassan** [comment]
@teotigraphix Good catch 👍 , I updated the wiki page [Defining-Custom-Views](https://github.com/codepath/android_guides/wiki/Defining-Custom-Views)
